setup/providers/kubernetes-v2/index.md

```diff
@@ -1,84 +1,71 @@
 ---
 layout: single
-title:  "Kubernetes V2 (Manifest-based)"
+title:  "Kubernetes Provider V2 (Manifest Based)"
 sidebar:
   nav: setup
 ---
 
 {% include toc %}
 
-> This is the second version of the Kubernetes provider in Spinnaker, not
-> necessarily support for version 2.0 of Kubernetes.
+The Spinnaker Kubernetes V2 provider fully supports manifest-based deployments. [Kubernetes provider V1](https://www.spinnaker.io/setup/providers/kubernetes/) is still supported.
 
-In Kubernetes, an [Account](/setup/providers/#accounts) maps to a
-credential able to authenticate against your desired Kubernetes Cluster.
+## Accounts
+
+For Kubernetes V2, a Spinnaker [Account](/setup/providers/#accounts) maps to a
+credential that can authenticate against your Kubernetes Cluster. Unlike with the V1 provider, in V2 the Account [does not map to Docker Registry Accounts](#migrating-from-the-v1-provider).
 
 ## Prerequisites
 
-The Kubernetes provider has two requirements, a
-[`kubeconfig`](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
-that you will
-likely have to fetch from your Kubernetes cluster's administrator, and
-[`kubectl`](https://kubernetes.io/docs/user-guide/kubectl/),
-which will be automatically be installed alongside Spinnaker for you in any
-Spinnaker packages we publish.
+The Kubernetes provider has two requirements:
 
-### kubeconfig
+* A [`kubeconfig`](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
 
-You must have a `kubeconfig` file that allows `kubectl` to authenticate against
-your cluster, as well as have read and write access against any resources you
-expect Spinnaker to manage. In the future, we will outline how to configure
-Kubernetes RBAC for Spinnaker to manage a restricted set of resources.
+    The `kubeconfig` allows `kubectl` to authenticate against your cluster and to have read/write access to any resources you expect
```

Comment:
I don't think this is worded right. It's common knowledge (for our target k8s users) what the relationship is. The goal here is to point out that you need your kubeconfig to have access to your cluster, and that access should extend to all resources you want Spinnaker to manage